pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/periodictask/ControllerPeriodicTask.java

```diff
@@ -83,7 +85,7 @@ protected final void runTask(Properties periodicTaskProperties) {
       }
 
       if (!tablesToProcess.isEmpty()) {
-        processTables(tablesToProcess);
+        processTables(tablesToProcess, taskParamsJson);
```

Comment:
Suggest passing the whole `Properties` into the `processTables` to be more flexible

Reply:
Ack